docs/modules/ROOT/pages/ios_faq.adoc

```diff
@@ -0,0 +1,49 @@
+= iOS Frequently Asked Questions (FAQ)
+:hardbreaks:
+:oauth2-app-url: https://marketplace.owncloud.com/apps/oauth2
+:toc: right
+
+Here you can find some of the most frequently asked questions about the ownCloud iOS app.
+
+== Usage
+
+=== Compatibility with ownCloud server older than version 10.0
+
+Previous ownCloud iOS app with support for ownCloud server < 10.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Compatibility with iOS versions older than iOS 12.0
+
+Previous ownCloud iOS app with support for iOS < 12.0 is still available in App Store:  
+https://apps.apple.com/app/id543672169
+
+=== Login with TOTP and other 2FA
+
+ownCloud server must have {oauth2-app-url}[the OAuth2 app] installed, configured, and enabled to use Two-Factor Authentication.
+Please contact your ownCloud administrator for more details.
+
+== Feature Requests
+
+== Access full folder via FileProvider, not only single files
+
+This seems like a bigger challenge for us. We track the status here:
+https://github.com/owncloud/ios-app/issues/604
+
+=== Text editing inside the iOS app
+
+Currently this isn't a planned feature. Focus is more to build the best integration in serious 3rd party text editing app.
+
+Here you can find previous discussion:
+https://github.com/owncloud/ios-app/issues/317
+
+=== Translation missing or bug
+
+You can help translate in your language or fix a bug.
+
+We use Transifex for translations. Please register for an account and join the global community:
+https://www.transifex.com/owncloud-org/owncloud/mobile-ios-app/
+
+=== Testing new features
+
+Join the beta program on iOS:
+https://owncloud.com/beta-testing/#ios
```

Comment:
pls use a macro (like the :oauth2-app-url:) for the link

Reply:
fixed with https://github.com/owncloud/ios-app/commit/4b31f225c301e9a952f3bd9dbed66d8878186328